File: programmers/level1/서울에서_김서방_찾기.java
package programmers;

public class 서울에서_김서방_찾기 {
	static
	class Solution {
		  public String solution(String[] seoul) {
		      String answer = "";
		      for(int i = 0; i < seoul.length; ++i) {
		    	  if(seoul[i].equals("Kim")) {
		    		  return "김서방은 "+i+"에 있다";
		    	  }
		      }
		      return answer;
		  }
		}
	public static void main(String[] args) {
		String[] seoul = {"Jane", "Kim"};
		Solution sol = new Solution();
		System.out.println(sol.solution(seoul));
	}
}

File: programmers/python/level1/서울에서_김서방_찾기.py
# def solution(seoul):
#     answer = ['김서방은 ', '', '에 있다']
#     answer[1] = str(seoul.index('Kim'))
#     return "".join(answer)

def solution(seoul):
    return "김서방은 {}에 있다".format(seoul.index('Kim'))

print(solution(["Jane", "Kim"]))